src/tools/spin/spinquic.cpp

```diff
@@ -927,6 +927,10 @@ main(int argc, char **argv)
         MsQuic->RegistrationClose(Registration);
         Registration = nullptr;
 
+        QuicSleep(4000);
```

Comment:
Remove sleep and maybe dump

Reply:
I'll remove the sleep, but keep the dump.